server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RabbitMQMailQueueTest.java

```diff
@@ -290,11 +291,6 @@ public void clearShouldNotFailWhenQueueIsEmpty() {
     public void clearShouldRemoveAllElements() {
     }
 
-    @Disabled("RabbitMQ Mail Queue do not yet implement getSize()")
```

Comment:
No import modification?

Reply:
There are other `@Disable` in this test. Then no import modification!